{"name": "test_invalid_login_shows_error", "status": "passed", "description": "Invalid login should display an error message.", "start": 1758000272604, "stop": 1758000294169, "uuid": "c718bcdd-c2ad-474c-856e-d4a1d15c0124", "historyId": "ef3403fe28d4b542f8ab987512d4fce5", "testCaseId": "100c00eabc4ddcc756ea44c0b5d967b8", "fullName": "tests.test_login#test_invalid_login_shows_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "DB-L-174"}, {"name": "thread", "value": "20960-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}